pkg/tfvars/libvirt/libvirt.go

```diff
@@ -7,7 +7,7 @@ import (
 	"net"
 
 	"github.com/apparentlymart/go-cidr/cidr"
-	"github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1alpha1"
+	"github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1beta1"
 	"github.com/pkg/errors"
 )
 
```

Comment:
You missed a `v1alpha1` in this file: https://github.com/openshift/installer/blob/master/pkg/tfvars/libvirt/libvirt.go#L23